labs-ios-starter/Login/View Controllers/LoginViewController.swift

```diff
@@ -30,13 +30,16 @@ class LoginViewController: DefaultViewController {
                                                object: nil,
                                                queue: .main,
                                                using: alertUserOfExpiredCredentials)
+
+        UIApplication.shared.open(ProfileController.shared.oktaAuth.identityAuthURL()!) { result in
```

Comment:
I can't figure out this part. Is this "hacky"? It's just calling that method?

Reply:
This is the same code that was in the login button previously. It's how Okta Auth was implemented by Spencer. We could probably take some time to figure out a way other than force-unwrapping it though